cmd/thanos/main.go

```diff
@@ -45,6 +45,7 @@ func main() {
 	logFormat := app.Flag("log.format", "Log format to use. Possible options: logfmt or json.").
 		Default(logging.LogFormatLogfmt).Enum(logging.LogFormatLogfmt, logging.LogFormatJSON)
 	tracingConfig := extkingpin.RegisterCommonTracingFlags(app)
+	reqLogConfig := extkingpin.RegisterRequestLoggingFlags(app)
```

Comment:
Do we need helper if we use it only once? 🤔 

(YAGNI rule) Not a blocker.

Reply:
Sure